src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/SQLCompiler.txt

```diff
@@ -0,0 +1,76 @@
+// These tests originated in the SQL compiler test suite.  
+// These all pass with C# too, so moved here instead for coverage for all backends and to ensure consistency.
+
+// IsError inside If
+>> If(true, IsError(1/0 + 1))
+true
+
+>> If(IsError(1/0), 1, 2)
+1
+
+>> Switch(IsError(1/0), true, 1, 2)
+1
+
+// If inside IsError
+>> IsError(If(false, 1/0, 1/0))
+true 
+
+>> IsError(If(true, 1/0, 1/0))
+true
+
+>> IsError(Switch(1, 1, 1/0, 1/0))
+true
+
+// IsError inside If inside IsError
+>> IsError(If(false, IsError(1/0), IsError(1/0)))
+false
+
+>> IsError(1/2 + 1) && IsError(1/0)
+false
+
+>> IsError(1/2 + 2)
+false
+
+>> IsError(1/0 + 1)
+true
+
+>> IfError(If(true, 1/0, 1/0), 2)
+2
+
+>> IsError(If(IfError(1/0 = 1, true), 1/0, 1/0))
+true
+
+// check string literals are sanitized
```

Comment:
This test doesn't make sense outside of SQL

Reply:
I'm not sure I understand.  I'm sure at one point this test was testing some bug in SQL codegen, but the test is valid in C# and passes?

Reply:
This test is to make sure that we don't interpret part of a literal string as a SQL Server command (https://xkcd.com/327/). Outside of SQL server, the test does pass, but it doesn't add anything to the coverage.

Reply:
Updated.  I removed those tests and will create a dedicated file I. The daraverse repo.